modules/serverless-ossm-installing-and-configuring-openshift-service-mesh-3-x.adoc

```diff
@@ -0,0 +1,243 @@
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-ossm-installing-and-configuring-openshift-service-mesh-3-x_{context}"]
+= Installing and configuring {SMProductShortName} 3.x
+
+[role="_abstract"]
+You can integrate {SMProductShortName} 3.x with {ServerlessProductShortName} by installing and configuring the required Istio components, gateways, and Knative Serving resources. Once these resources are configured, you can deploy the Knative Serving instance with Istio enabled to ensure that your serverless workloads run within the {SMProductShortName} environment. 
+
+.Procedure
+
+. Create an `Istio` resource in the `istio-system` namespace with the following configuration:
++
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  values:
+    meshConfig:
+      defaultConfig:
+        terminationDrainDuration: 35s
+  updateStrategy:
+    inactiveRevisionDeletionGracePeriodSeconds: 30
+    type: InPlace
+  namespace: istio-system
+  version: v1.26-latest
+----
+
+. Apply the resource by running the following commands:
++
+[source,terminal]
+----
+$ oc new-project istio-system
+----
++
+[source,terminal]
+----
+$ oc apply -f istio.yaml
+----
+
+. Create an `IstioCNI` resource in the `istio-cni` namespace with the following configuration:
++
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1
+kind: IstioCNI
+metadata:
+  name: default
+spec:
+  namespace: istio-cni
+  version: v1.26-latest
+----
+
+. Apply the resource by running the following commands:
++
+[source,terminal]
+----
+$ oc new-project istio-cni
+----
++
+[source,terminal]
+----
+$ oc apply -f istio-cni.yaml
+----
+
+. Create a `Gateway Deployment` resource in the `knative-serving-ingress` namespace with the following configuration:
++
+[source,terminal]
+----
+$ oc new-project knative-serving-ingress
```

Comment:
as Serverless should be installed at this point, the ns already exists. Please delete this step

Reply:
Removed the step